core/src/main/java/de/danoeh/antennapod/core/widget/WidgetUpdater.java

```diff
@@ -99,7 +98,7 @@ public static void updateWidget(Context context, WidgetState widgetState) {
                 try {
                     icon = Glide.with(context)
                             .asBitmap()
-                            .load(ImageResourceUtils.getFallbackImageLocation(widgetState.media))
+                            .load(widgetState.media.getImageLocation())
```

Comment:
Same here, fallback is now the same as main. Please have a look at what the code did differently before. I currently don't remember.

Reply:
Yes, see comment above.